src/app/src/main/kotlin/org/imperial/mrc/hint/controllers/BaselineController.kt

```diff
@@ -48,4 +48,14 @@ class BaselineController(private val fileManager: FileManager,
         return getAndValidate(FileType.Population)
     }
 
+    @GetMapping("/validate/")
+    @ResponseBody
+    fun validate(): ResponseEntity<String> {
+        return apiClient.validateBaselineCombined(
+                fileManager.getFile(FileType.PJNZ),
```

Comment:
Ah ok - yeah the reason I didn't use getFiles was because I'd then have to extract the values from the returned map by string key to construct the json to pass to hintr, which seemed more of a faff. But if we can just serialize it straight through that'll save some code.

Reply:
There is also `mapValues` kotlin function which transforms one map into another with the same keys just by transforming the values: https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/map-values.html

Reply:
Actually, it turned out that another issue is that getFiles misses out the entry for missing files, while the validate baseline endpoint required all three properties to be present, so I have modified this approach slightly.

Reply:
Ah of course